R/utils-shared.R

```diff
@@ -35,6 +35,70 @@ startRender <- function(data, output, author, reportName){
 
 ############ used in dvhydrograph-data, fiveyeargwsum-data, uvhydrograph-data ############ 
 
+getGroundWaterLevels<- function(ts, ...){
+  y <- as.numeric(ts$gwlevel[['groundWaterLevel']])
+  x <- ts$gwlevel[['dateString']]
+  time = as.POSIXct(strptime(x, "%Y%m%d"))
+  month <- format(time, format = "%y%m") #for subsetting later by month
+  return(data.frame(time=time, value=y, month=month, stringsAsFactors = FALSE))
+}
```

Comment:
Let's also wait on that, we need to find a sprint to elevate all of our code improvement tasks and do them.